nio/responses.py

```diff
@@ -831,6 +838,33 @@ def from_dict(
         return cls(parsed_dict, event_type, state_key, room_id)
 
 
+class RoomGetEventResponse(Response):
+    """A response indicating successful room get event request.
+
+    Attributes:
+        event (Event): The requested event.
+    """
+
+    event: Event = field()
+
+    @staticmethod
+    def create_error(parsed_dict):
+        return RoomGetEventError.from_dict(parsed_dict)
+
+    @classmethod
+    def from_dict(
+        cls,
+        parsed_dict: Dict[str, Any]
+    ) -> Union["RoomGetEventResponse", RoomGetEventError]:
+        if "errcode" in parsed_dict:
```

Comment:
Oh wait, you aren't verifying the response schema. We usually add a json schema to `schema.py` and use the `verify` decorator instead of manually checking for the `errcode` in the dict.

Reply:
I took the room_event schema which should work fine.